client-app/src/Components/StatusDisplayPage.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react'; // Do we need this file anymore?
```

Comment:
we should stash this file after commiting in local , and send only the file which has been changed.